scripts/example-pipelines-test.sh

```diff
@@ -1,3 +1,150 @@
 #!/bin/bash
 
-echo "Temporary test file"
+cp -L $KUBECONFIG /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig
+export IMG=${CI_OPERATOR_IMG}
+
+winImageDownloadURL="http://http-server/disk.img"
+
+# Deploy resources
+echo "Deploying resources"
+./scripts/deploy-resources.sh
+
+# SECRET
+key="/tmp/secrets/accessKeyId"
+token="/tmp/secrets/secretKey"
+
+if test -f "$key" && test -f "$token"; then
+  id=$(cat $key | tr -d '\n' | base64)
+  token=$(cat $token | tr -d '\n' | base64 | tr -d ' \n')
+
+  oc apply -n kubevirt -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tekton-operator-container-disk-puller
+  namespace: kubevirt
+type: Opaque
+data:
+  accessKeyId: "${id}"
+  secretKey: "${token}"
+EOF
+fi
+
+echo "Create config map for http server"
```

Comment:
IMO it would make more sense to move th creation of the ConfigMap after the creation of the DV.